meshtastic/interfaces/ble/receive_service.py

```
@@ -0,0 +1,419 @@
+
+    @staticmethod
+    def _handle_read_loop_disconnect(
+        iface: "BLEInterface", error_message: str, previous_client: BLEClient
+    ) -> bool:
+        """Return whether receive loop should continue after a disconnect."""
+        logger.debug("Device disconnected: %s", error_message)
+        should_continue = iface._handle_disconnect(
+            f"read_loop: {error_message}", client=previous_client
+        )
+        if not should_continue:
+            iface._set_receive_wanted(False)
```

Ruff (0.15.5)

[warning] Boolean positional value in function call

(FBT003)